Cmder.bat

```diff
@@ -1,20 +1,66 @@
 @echo off
+
 SET CMDER_ROOT=%~dp0
 
-:: Remove Trailing '\'
-@if "%CMDER_ROOT:~-1%" == "\" SET CMDER_ROOT=%CMDER_ROOT:~0,-1%
-
-if not exist "%CMDER_ROOT%\config\user_ConEmu.xml" (
-    if not exist "%CMDER_ROOT%\config" mkdir "%CMDER_ROOT%\config" 2>nul
-    copy "%CMDER_ROOT%\vendor\ConEmu.xml.default" "%CMDER_ROOT%\config\user_ConEmu.xml" 1>nul
-    if %errorlevel% neq 0 (
-        echo ERROR: CMDER Initialization has Failed
-        exit /b 1
-    )
+set CMDER_TERMINAl=conemu
```

Comment:
Please change the `CMDER_TERMINAl` to `CMDER_TERMINAL` (notice the lower case `l` at the end)

Reply:
Done